tests/globalhelper/deployment.go

```diff
@@ -32,6 +33,18 @@ func IsDeploymentReady(operatorNamespace string, deploymentName string) (bool, e
 	return false, nil
 }
 
+// IsDeploymentInstalled checks if deployment is installed.
+func IsDeploymentInstalled(
+	cs *client.ClientSet, operatorNamespace string, operatorDeploymentName string) (bool, error) {
```

Comment:
Why do you need cs in func parameters ?

Reply:
ah that's leftover from the inspiration code, removing